### ISSUE TITLE
Add a Buildpoint based generator.

### ISSUE THREAD

**Issue by UrsZeidler** [UrsZeidler/shr5rcp#11]
There seems the need of an alternative generator system based on BP (Buildpoint). 
The parts are analog to the priority parts of the build system but they have a value instead of the priority.

some community thoughts :

http://shadowrun.com/forums/discussion/38135/messing-with-the-priorities-system#Item_8
http://forums.shadowruntabletop.com/index.php?topic=10206.0   TL;DR

**Comment by NoxMortem** [comment]
The Run&Gun provides several new generators which will require updates. As some of those work substantially different this might need a lot of work.

**Comment by UrsZeidler** [comment]
Yeah I saw this, I also looked at resources some guys doing for it , http://forums.shadowruntabletop.com/index.php?topic=19033.75
